### PR TITLE
䷓  Template for navigation bar

### LINKED ISSUE #4: Crear estructura Tab Bar

### PR DESCRIPTION
**Pull request by yiperu**
Make template for show navigation bar

### CODE REVIEW

File: app/src/main/java/com/arqoders/hotelapp/Details/DetailsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.arqoders.hotelapp.databinding.FragmentDetailsBinding
+
+
+class DetailsFragment : Fragment() {
+    // TODO: Rename and change types of parameters
```
Comment (marcossalto) [inline app/src/main/java/com/arqoders/hotelapp/Details/DetailsFragment.kt]: Remove comments

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,12 @@
 <resources>
     <string name="app_name">HotelApp</string>
+    <string name="home">Inicio</string>
+    <string name="details">Detalles</string>
+    <string name="cache">Cache</string>
+    <string name="preference">Preferencia</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (yxzandra) [inline app/src/main/res/values/strings.xml]: remove this string unused

File: app/src/main/res/layout/fragment_cache.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@android:color/holo_green_light"
+    tools:context=".Cache.CacheFragment">
+
+    <!-- TODO: Update blank fragment layout -->
```
Comment (marcossalto) [inline app/src/main/res/layout/fragment_cache.xml]: Remove comment

File: app/src/main/res/layout/fragment_details.xml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@android:color/holo_blue_light"
+    tools:context=".Details.DetailsFragment">
+
+    <!-- TODO: Update blank fragment layout -->
```
Comment (marcossalto) [inline app/src/main/res/layout/fragment_details.xml]: Remove comment

File: app/src/main/res/layout/fragment_preference.xml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@android:color/holo_purple"
+    tools:context=".Preference.PreferenceFragment">
+
+    <!-- TODO: Update blank fragment layout -->
```
Comment (marcossalto) [inline app/src/main/res/layout/fragment_preference.xml]: Remove comment

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -1,3 +1,12 @@
 <resources>
     <string name="app_name">HotelApp</string>
+    <string name="home">Inicio</string>
+    <string name="details">Detalles</string>
+    <string name="cache">Cache</string>
+    <string name="preference">Preferencia</string>
+    <!-- TODO: Remove or change this placeholder text -->
```
Comment (marcossalto) [inline app/src/main/res/values/strings.xml]: Remove comment

**Review by marcossalto — CHANGES_REQUESTED** [review]

**Review by yxzandra — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/arqoders/hotelapp/Details/DetailsFragment.kt (hunk 1 above)
Comment (yiperu) [reply app/src/main/java/com/arqoders/hotelapp/Details/DetailsFragment.kt]: ok, removed

File: app/src/main/res/layout/fragment_cache.xml (hunk 3 above)
Comment (yiperu) [reply app/src/main/res/layout/fragment_cache.xml]: ok, removed

File: app/src/main/res/layout/fragment_details.xml (hunk 4 above)
Comment (yiperu) [reply app/src/main/res/layout/fragment_details.xml]: ok, removed

File: app/src/main/res/layout/fragment_preference.xml (hunk 5 above)
Comment (yiperu) [reply app/src/main/res/layout/fragment_preference.xml]: ok, removed

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (yiperu) [reply app/src/main/res/values/strings.xml]: ok, removed

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (yiperu) [reply app/src/main/res/values/strings.xml]: ok, removed

**Pull request merged**